### ISSUE TITLE
Add

### ISSUE THREAD

**Issue by rt87** [jc21/nginx-proxy-manager#1726]
**Is your feature request related to a problem? Please describe.**
Only the problem of me messing around with my various services and screwing up occassionally. This leads to the proxy not being able to do its job due to my reckless changes to firewall rules, port or host changes etc, etc.

**Describe the solution you'd like**

The possibility to add a "periodic healthcheck" to hosts (either all or individually) as a sort of healthcheck. The check should basically send a request to the target host (without any location) and expect a http status of 200 (or 200-ish / "OK"; redirects may be acceptable as well, depending on the use case). If the OK-condition is met, some OK-symbol (green circle, check mark, ...) should be visible from the resp. overview page. If not, a you-screwed-up symbol (red circle, red X, ...) should be displayed instead.
**Note:** Of course this does not guarantee that each and every location, that I may need in the future, exists and is valid, but at least I would know that my hostname exists and is http-reachable under the given port.

**Describe alternatives you've considered**
I could of course either write a cron script which curls all the hosts or simply use service that is meant for such checks, e.g. healthchecks.io. However, a somehwat major drawback would be that I would have to keep both NPM and my healthchecks synchronized, which may get tedious in the long run....

**Additional context**
None

**Comment by brink668** [comment]
I agree with this, Status shows "Online" its misleading. Purposely put in a service that does not exist and it still shows it as "Online"

**Comment by TheUntouchable** [comment]
I just started using npm and was really confused about the "online" status. I played with some commercial proxys in the past already and most of them have such health checks build in by default and I thought it would be the same "online" thing here :D

**Comment by wj7998** [comment]
I am also confused on "Online" status, in my use case, online status should indicate forwarded port should be active on listening. I would recommend to have following status,
"setting error" - with ! yellow circle
"setting right, but forward port is unreachable" - with red circle
"setting right, but forward port is reachable" - with green circle
